gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -82,6 +85,20 @@ def unconfigure_httpd_application
       end
 
       #
+      # Kerberos KRB5 File Methods
+      #
+      def enable_kerberos_dns_lookups
+        updated_content = File.readlines(KERBEROS_CONFIG_FILE).collect do |line|
+          if line =~ /#{KERBEROS_DNS_LOOKUP}/
+            line.gsub('false', 'true')
+          else
+            line
+          end
+        end
+        File.open(KERBEROS_CONFIG_FILE, "w") { |f| updated_content.each { |line| f.puts line } }
+      end
```

Comment:
After discussing with @abellotti, in an effort to avoid unnecessary complexity, we have decided that we will not save a backup.timestamp copy and not take a status value and just set the new value to `true`